src/nss/inc_loot.nss

```diff
@@ -1400,7 +1419,8 @@ float GetAverageLootValueOfItem(int nTier, string sType)
                 object oTest = GetFirstItemInInventory(oChest);
                 while (GetIsObjectValid(oTest))
                 {
-                    int nBaseType = GetBaseItemType(oTest);
+                    //int nBaseType = GetBaseItemType(oTest);
+                    /* I don't think we need to do this anymore, every magic ammo and throwing weapon are a stack size of 1
```

Comment:
I don't know for sure that GetItemGoldValue is correct in this case!
I remember having a hard time with boomerang items and values, but that was because of ILRStackSize etc.
With all the 2da changes I'm not certain exactly what will happen and whether this needs adjusting or not

Reply:
With the 2da change it means that a +1 throwing axe that should be valued at 2000 gold can just use that value without any special calculations

We just need to ensure that the stack size is 1 at all times, or the numbers will be really off